LayoutTests/imported/w3c/web-platform-tests/digital-credentials/identity-get.tentative.https.html

```diff
@@ -0,0 +1,195 @@
+</body>
+<script type="module">
+  import { makeGetOptions, sendMessage, loadIframe } from "./support/helper.js";
+
+  const iframeSameOrigin = document.querySelector("iframe#same-origin");
+  const iframeCrossOrigin = document.querySelector("iframe#cross-origin");
+
+  promise_setup(async () => {
+    const hostInfo = get_host_info();
+    await Promise.all([
+      loadIframe(
+        iframeCrossOrigin,
+        `${hostInfo.HTTPS_REMOTE_ORIGIN}/digital-credentials/support/iframe.html`
+      ),
+      loadIframe(iframeSameOrigin, "/digital-credentials/support/iframe.html"),
+    ]);
+  });
+
+  promise_test(async (t) => {
+    iframeSameOrigin.focus();
+    for (const global of [window, iframeSameOrigin.contentWindow]) {
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get()
+      );
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({})
+      );
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({ x: "y" })
+      );
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({ x: "y", y: "z" })
+      );
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({ mediation: "required" })
+      );
+
+      const abortController = new AbortController();
+      const { signal } = abortController;
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({ signal })
+      );
+
+      await promise_rejects_dom(
+        t,
+        "NotSupportedError",
+        global.DOMException,
+        global.navigator.identity.get({ signal, mediation: "required" })
+      );
+    }
+  }, "Calling navigator.identity.get() without an digital member same origin.");
+
+  promise_test(async (t) => {
+    for (const provider of [undefined, []]) {
+      const options = makeGetOptions(provider);
+      await test_driver.bless("user activation");
+      await promise_rejects_js(t, TypeError, navigator.identity.get(options));
+    }
+  }, "navigator.identity.get() API rejects if there are no providers.");
+
+  promise_test(async (t) => {
+    iframeSameOrigin.focus();
+    const { contentWindow: iframeWindow } = iframeSameOrigin;
+    for (const provider of [undefined, []]) {
+      const options = makeGetOptions(provider);
+      await test_driver.bless("user activation");
+      await promise_rejects_js(
+        t,
+        iframeWindow.TypeError,
+        iframeWindow.navigator.identity.get(options)
+      );
+    }
+  }, "navigator.identity.get() API rejects if there are no providers for same-origin iframe.");
```

Comment:
It looks like we're consistently failing this test on iOS simulator! Given the exception thrown, I believe `iframeSameOrigin.focus()` isn't behaving as expected?

```
+FAIL navigator.identity.get() API rejects if there are no providers for same-origin iframe. promise_rejects_js: function "function() { throw e }" threw object "NotAllowedError: The document is not focused." ("NotAllowedError") expected instance of function "function TypeError() {
+    [native code]
+}" ("TypeError")
```

Reply:
Alternative to fixing is to file a follow up FIXME bugzilla and check-in the failing expectations for iOS, I suppose.

Reply:
>  I believe iframeSameOrigin.focus() isn't behaving as expected?

That's correct. There's some issues with focus()'ing iframes in webkit generally, I believe. I've seen people trying to do things like this:
https://gist.github.com/erkobridee/e567d22180b9118229441f71791f6ec7

I couldn't find bug for it, so might just file one.